config.toml

```
@@ -490,3 +496,12 @@ logo = "snyk.png"
 [[params.supportedby]]
 url = "https://vexxhost.com"
 logo = "vexxhost.png"
+
+# "Commercial support" section on the support page
+[[params.vendors]]
+url = "https://docs.microsoft.com/en-us/azure/aks/keda-about"
+logo = "microsoft.png"
+
+[[params.vendors]]
+url = "https://cloud.redhat.com/blog/custom-metrics-autoscaler-on-openshift"
+logo = "red-hat.png"
```

`vendor` might be a bit unclear, maybe this is better?
```suggestion
[[params.managedOfferings]]
url = "https://docs.microsoft.com/en-us/azure/aks/keda-about"
logo = "microsoft.png"

[[params.managedOfferings]]
url = "https://cloud.redhat.com/blog/custom-metrics-autoscaler-on-openshift"
logo = "red-hat.png"
```

---

I think that vendor is ok, since it is just internal stuff for rendering the page.

---

Fine by me